### PR TITLE
Added Wearable-Actions to Notifications

### PR DESCRIPTION
**Pull request by chr1shaefn3r**
To better support wearable devices, I also added the notification actions via the "WearableExtender()" to the NotificationCompat.Builder.

This should give all the Android Wear-compatible device owner the possibility to mark incoming messages as read without the need to pull out their phone, aka. enable maximum laziness (but why else would you own a smart watch, right?).

Tested it with my own Pebble and worked great.

### CODE REVIEW

**Comment by mcginty** [comment]
Can you confirm that action is really missing on `HEAD` of master? I just tried it with a Wear emulator and it looks like it's already available even without this change.

**Comment by chr1shaefn3r** [comment]
Yes, I can confirm that the actions is really missing on HEAD of master on my Pebble before this change.
Maybe this is more of an "Added Pebble support" PR then.

File: src/org/thoughtcrime/securesms/notifications/MessageNotifier.java [hunk 1]
```diff
@@ -196,8 +196,11 @@ private static void sendSingleThreadNotification(Context context,
     if (recipient.getContactUri() != null) builder.addPerson(recipient.getContactUri().toString());
 
     if (masterSecret != null) {
-      builder.addAction(R.drawable.check, context.getString(R.string.MessageNotifier_mark_as_read),
-                        notificationState.getMarkAsReadIntent(context, masterSecret));
+      NotificationCompat.Action markAsReadAction =
+        new NotificationCompat.Action.Builder(R.drawable.check, context.getString(R.string.MessageNotifier_mark_as_read),
+                                              notificationState.getMarkAsReadIntent(context, masterSecret)).build();
```
Comment (moxie0) [inline src/org/thoughtcrime/securesms/notifications/MessageNotifier.java]: looks to me like there's no reason to use a builder here

File: src/org/thoughtcrime/securesms/notifications/MessageNotifier.java [hunk 2]
```diff
@@ -246,8 +249,11 @@ private static void sendMultipleThreadNotification(Context context,
     builder.setDeleteIntent(PendingIntent.getBroadcast(context, 0, new Intent(DeleteReceiver.DELETE_REMINDER_ACTION), 0));
 
     if (masterSecret != null) {
-      builder.addAction(R.drawable.check, context.getString(R.string.MessageNotifier_mark_all_as_read),
-                        notificationState.getMarkAsReadIntent(context, masterSecret));
+       NotificationCompat.Action markAllAsReadAction =
+         new NotificationCompat.Action.Builder(R.drawable.check, context.getString(R.string.MessageNotifier_mark_all_as_read),
+                                               notificationState.getMarkAsReadIntent(context, masterSecret)).build();
```
Comment (moxie0) [inline src/org/thoughtcrime/securesms/notifications/MessageNotifier.java]: same

File: src/org/thoughtcrime/securesms/notifications/MessageNotifier.java (hunk 1 above)
Comment (chr1shaefn3r) [inline src/org/thoughtcrime/securesms/notifications/MessageNotifier.java]: true that, will fix.

File: src/org/thoughtcrime/securesms/notifications/MessageNotifier.java (hunk 2 above)
Comment (chr1shaefn3r) [inline src/org/thoughtcrime/securesms/notifications/MessageNotifier.java]: here too

**Comment by midi** [comment]
Thank you @chr1shaefn3r  
I noticed that the reply action is missing, could you add that?
It is super useful.

Another issue: #1699

**Comment by chr1shaefn3r** [comment]
@midi you are welcome.

I'm working on porting @Karalix's PR (SMSSecure#135) to the TextSecure code base.
If I don't run in any major problem, expect a PR  around next week.

**Pull request closed without merging**